libqtile/core/manager.py

```diff
@@ -197,6 +198,16 @@ def _configure(self):
 
         hook.fire("startup_complete")
 
+    @property
+    def display_name(self):
+        # Dots might appear in the host part of the display name
+        # during remote X sessions. Let's strip the host part first
+        name = self.core.display_name
+        display_number = name.partition(":")[2]
+        if "." not in display_number:
+            name += ".0"
```

Comment:
Could you move this part into the core? It is X specific

Reply:
I'll need to move it and add an ipc_socket_path property to core as well, is that ok? My reasoning is that when we have a separate core there will be a separate way of naming the socket for the ipc server now managed by Qtile.

Reply:
Actually thinking about it again, this string manipulation is only required when finding the socket file, _not_ when we are using the display of `Qtile` (necessarily). I.e. it is also used by command clients to communicate with `Qtile`. So maybe just put it as a standalone function like it was originally but inside `libqtile/backend/x11/__init__.py` or somewhere like that.

Reply:
RE irc: I'll take a look at how the command clients do things and try to emulate them while I try to move this logic into the backend.

Reply:
Actually I think you can completely remove this property and just use `self.core.display_name` on line 211. The lines 223-225 are redundant as they are also within `ipc.find_sockfile`. If we need to change this in the future due to new cores then we can deal with it if/when we need to.